#include "receiptwindow.h"
#include "ui_receiptwindow.h"

receiptwindow::receiptwindow(QWidget *parent) :
    QDialog(parent),
    ui(new Ui::receiptwindow)
{
    ui->setupUi(this);
}

receiptwindow::~receiptwindow()
{
    delete ui;
}

void receiptwindow::receiveUpdatedSaldoFromMainWindow(QString maara,QString updatedsaldo)
{

    qDebug()<<"Nyt on vastaanotettu p채ivitetty saldo mainwindow.cpp:lt채."<<updatedsaldo;
    ui->labelSumma->setText(maara);
    ui->labelSaldo->setText(updatedsaldo);

}

void receiptwindow::receiveErrorMessageFromExe()
{
    qDebug()<<"Nyt on errorit exelt채 vastaanotettu, ei oo rahhaa!";
    ui->labelError->setText("Kate ei riit채!");
    ui->labelSaldo->setText("Error");
    ui->labelSumma->setText("Error");

}
